client/state/selectors/test/get-blog-stickers.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getBlogStickers } from '../';
+
+describe( 'getBlogStickers()', () => {
+	it( 'should return stickers for a known blog', () => {
+		const stateIn = {
```

Comment:
I try to stick to the same naming scheme every time. Something like:
1. `prevState = ...`
2. `nextState = reduce( prevState, action )`
3. `expectedState = .....`
4. expect( nextState ).eql( expectedState )